src/material-experimental/mdc-button/harness/button-harness.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness, HarnessPredicate} from '@angular/cdk-experimental/testing';
+import {coerceBooleanProperty} from '@angular/cdk/coercion';
+import {ButtonHarnessFilters} from './button-harness-filters';
+
+
+/**
+ * Harness for interacting with a standard mat-button in tests.
+ * @dynamic
+ */
+export class MatButtonHarness extends ComponentHarness {
+  static hostSelector = [
```

Comment:
There's an array of these defined as `BUTTON_HOST_ATTRIBUTES`. Could we export and re-use it here? Otherwise we'll have to remember to update this array if we add more button types.

Reply:
I really just want to change this to use e.g. `.mat-button-base`, which was in another PR when I wrote this. Added a `TODO`  for now